aim/sdk/objects/plotly.py

```diff
@@ -0,0 +1,33 @@
+from aim.sdk.num_utils import inst_has_typename
+from aim.storage.object import CustomObject
+from aim.storage.types import BLOB
+
+
+@CustomObject.alias("aim.plotly")
+class Plotly(CustomObject):
+    AIM_NAME = "aim.plotly"
+
+    def __init__(self, obj):
+        super().__init__()
+
+        if inst_has_typename(obj, ["Figure", "BaseFigure"]):
+            self._prepare(obj)
+        else:
+            raise TypeError("Object is not a Plotly instance")
+
+    def _prepare(self, obj):
+        assert hasattr(obj, "to_json")
+
+        self.storage["data"] = BLOB(data=obj.to_json())
```

Comment:
please add the `source='plotly'` and `format='raw_json' to storage to make this future-proof.once we change format and/or add new types of input, this will be required.